packages/cloud-sync-server/src/dto/sync.dto.ts

```
@@ -0,0 +1,147 @@
+  pwdHash: string;
+
+  @Rule(getSchema(PrimeSyncUploadItem).optional())
+  lock?: PrimeSyncUploadItem;
+
+  @Rule(
+    RuleType.array()
+      .items(getSchema(PrimeSyncUploadItem))
+      .when('pwdHash', {
+        is: RuleType.string().not('').required(),
+        then: RuleType.array().min(1),
+        otherwise: RuleType.required(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)